src/hacking/building-for-openharmony.md

```diff
@@ -14,6 +14,7 @@ Building for OpenHarmony requires the following:
 ### Setting up the OpenHarmony SDK
 
 The OpenHarmony SDK is required to compile applications for OpenHarmony.
+The minimum version of SDK that Servo currently supports is v5.0.
```

Comment:
```suggestion
The minimum version of SDK that Servo currently supports is v5.0 (API-12).
```

It appears that in the next days API level 13 will be released (currently in beta) and that one will also be `5.0`, with just the patch version increased. So I guess we need should both document the version number and the api level